applications/stepswitch/src/stepswitch_local_extension.erl

```diff
@@ -177,19 +190,46 @@ handle_info(_Info, State) ->
 -spec handle_event(kz_json:object(), state()) -> {'reply', []}.
 handle_event(JObj, #state{request_handler=RequestHandler
                          ,resource_req=Request
+                         ,call_id=CallId
                          }) ->
-    case kapps_util:get_event_type(JObj) of
-        {<<"error">>, _} ->
+    case get_event_type(JObj) of
+        {<<"error">>, _, _} ->
             <<"bridge">> = kz_json:get_value([<<"Request">>, <<"Application-Name">>], JObj),
             lager:debug("channel execution error while waiting for execute extension: ~s"
                        ,[kz_util:to_binary(kz_json:encode(JObj))]),
             gen_listener:cast(RequestHandler, {'local_extension_result', local_extension_error(JObj, Request)});
-        {<<"call_event">>, <<"CHANNEL_DESTROY">>} ->
-            gen_listener:cast(RequestHandler, {'local_extension_result', local_extension_success(Request)});
-        {<<"call_event">>, <<"CHANNEL_EXECUTE_COMPLETE">>} ->
+        {<<"call_event">>, <<"CHANNEL_TRANSFEREE">>, _} ->
+            Transferee = kz_call_event:other_leg_call_id(JObj),
+            gen_listener:cast(RequestHandler, {'replaced', Transferee}),
+            gen_listener:add_binding(RequestHandler, ?CALL_BINDING(Transferee));
+        {<<"call_event">>, <<"CHANNEL_TRANSFEROR">>, _} ->
```

Comment:
@lazedo Tracking replaced makes sense but why treat the other leg of the transferee/transferor as the stepswitch_local_extension.

For example, if we a local extension A -> B was created,  then transferred such that A -> C we should track A as the local extension.  However, if we receive the transferee event (for A) and take the other leg (C) then the local extension would then be tracking C, no?

For the transferor situation, if A -> B and A preforms the transfer such that B -> C then the original local extension (again A) has been destroyed.  Why would local extension track B in this case?

(Please note these are from  my head I will need to deploy this branch to confirm... just surprised that the replaced event was not sufficient)